components/Column.js

```diff
@@ -1,7 +1,20 @@
 import Row from './Row'
+import { Component } from 'react';
 
-export default ({ children, heading, loading, loadingAmount = 20, className }) => (
-  <div className={`column ${className}`}>
+class Column extends Component {
+  constructor (props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    Array.prototype.forEach.call(document.querySelectorAll('.column .active'), activeRow => {
+      activeRow.scrollIntoView();
```

Comment:
wow hah, I've never even heard of this function. very cool!

Reply:
We should maybe scope this `.column .active` to this particular column otherwise it'll call it on every column on the screen. Can be done by adding a unique id to each column.

Reply:
You are totally right. I'll try to include this next time I'm making changes.